src/Components/CardSection/styles.js

```diff
@@ -0,0 +1,20 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  containerStyle: {
+    borderBottomWidth: 1,
+    padding: 5,
+    backgroundColor: '#FFF',
+    justifyContent: 'flex-start',
```

Comment:
You probably not need justifyContent: 'flex-start', as it is the default.

Same for position relative.